tests/flow_error.rs

```diff
@@ -53,7 +53,9 @@ macro_rules! test_cases {
 
                 match parse(&contents) {
                     Err(e) => {
-                        let got = Dumb::error_to_string(&e)?;
```

Comment:
Why do we spell out the highlighter generation?

Reply:
What exactly do you mean?

Reply:
this seems to unroll `error_to_string` and I wondered why

Reply:
It covers the uncovered function `Deploy::format_error_with`. Sneaky, i know!

Reply:
alternatively, we could just `format_error_with`? it seems to be a copy of `format_error`. This feels like a harmful example of code for coverage's sake :(

Reply:
`format_error_with`, as far as I can tell, is never called anywhere so removing it seems the right thing to do.